src/core/mac/mac_frame.cpp

```diff
@@ -59,13 +59,19 @@ void Frame::InitMacHeader(Type             aType,
                           const Addresses &aAddrs,
                           const PanIds    &aPanIds,
                           SecurityLevel    aSecurityLevel,
-                          KeyIdMode        aKeyIdMode)
+                          KeyIdMode        aKeyIdMode,
+                          bool             aSuppressSequence)
```

Comment:
Do we currently have any use cases for `InitMacHeader()` where we would want to set `aSuppressSeq`?

Should we keep it the same as before (and add support for it only when we need it, to avoid its size overhead)?